### PR TITLE
changes up some chaplain weapons

### PR DESCRIPTION
**Pull request by ghost**
a lot of the chaplain weapons only looked different

tada now they're all unique mostly kinda

#### Changelog

:cl:
rscadd: Chaplain weapons are now more varied in effects.
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
I thought there would be something unique, all you did was buff the damage on stuff and make the dark blade by far the best chap weapon. :-1:

**Comment by utahclock** [comment]
Again you used online editor..you have not tested this

**Comment by ghost** [comment]
utahclock you fucking moron are you absolutely fucking retarded have you listened to anything i've ever fucking said you fucking doofus jesus cfucking christ

# I CAN'T USE ANYTHING BUT

**Comment by ghost** [comment]
@ShadowDeath6 how is the dark blade even good

also the hammer hammers people now

**Comment by utahclock** [comment]
If you can download and play on byond. You can use dream maker. And can use a git program.

**Comment by ShadowDeath6** [comment]
Hammer will hit people into walls which will stun and fuck their skull
Dark blade is a claymore with higher block chance which makes it way better considering sharpener.dm

**Comment by ghost** [comment]
i don't have

github as a program

and ive explained why

**Comment by utahclock** [comment]
You dont have to have github to test and compile your changes in dm..

**Comment by ShadowDeath6** [comment]
Adam you shouldn't be making changes if you can't do them without help, people are spending time and effort looking for and fixing errors in your PRs and it's probably getting tiring.

**Comment by ghost** [comment]
Iiiiii'm looking for the errors myself

File: code/game/objects/items/weapons/holy_weapons.dm [hunk 1]
```diff
@@ -273,30 +316,40 @@
...
-	armour_penetration = 35
+	force = 15
+	armour_penetration = 70
 	slot_flags = SLOT_BACK
 	sharpness = IS_SHARP
 	attack_verb = list("chopped", "sliced", "cut", "reaped")
+	
+/obj/item/weapon/nullrod/scythe/attack_self(mob/user)
+	if(user.mind && (user.mind.assigned_role == "Chaplain"))
+		user.add_atom_colour(#000000)
```
Comment (monster860) [inline code/game/objects/items/weapons/holy_weapons.dm]: The `#000000` needs to have quotes around it. You know, those things that usually go around pieces of text.

**Review by monster860 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weapons/holy_weapons.dm (hunk 1 above)
Comment (ShadowDeath6) [reply code/game/objects/items/weapons/holy_weapons.dm]: :triumph:

Comment (monster860) [reply code/game/objects/items/weapons/holy_weapons.dm]: One day, while Andy was coding, Woody got fucking pissed. He could no longer help himself! Woody watched as Andy screwed up his code. He approached andy which startled him and made him throw junk merge commits everywhere, and on Woody too.

Comment (ghost) [reply code/game/objects/items/weapons/holy_weapons.dm]: i'm saving that

**Comment by ghost** [comment]
Rude, i was fixin it

**Comment by Super3222** [comment]
@AdamElTablawy There's a way to make hammers have knockback, but without slamming people into a wall. I think you should take it because a null rod option that can do 30 consistent damage will trump every other type of null rod.

**Pull request closed without merging**